src/RequestValidationException.php

```diff
@@ -5,26 +5,27 @@
 namespace Yiisoft\RequestModel;
 
 use RuntimeException;
+use Generator;
 
 final class RequestValidationException extends RuntimeException
 {
     private const MESSAGE = 'Request model validation error';
-    private array $errors;
+    private Generator $errors;
 
-    public function __construct(array $errors)
+    public function __construct(Generator $errors)
     {
         $this->errors = $errors;
         parent::__construct(self::MESSAGE);
     }
 
     public function getErrors(): array
     {
-        return $this->errors;
+        return iterator_to_array($this->errors);
```

Comment:
Using generator forbids to call `getErrors` more than once.

Reply:
the generator will be removed